pandas/core/groupby.py

```diff
@@ -252,6 +252,11 @@ def __init__(self, key=None, level=None, freq=None, axis=0, sort=False):
         self.indexer = None
         self.binner = None
 
+    # _attributes is used in __repr__below
```

Comment:
this is just not needed at all

Reply:
_attributes should be a *list* of the actual attributes, NOT attached to the default values.

Reply:
this is *already* done for the resampler to produce the signature, just follow the example.